docs/beginner/tutorial2-surface/README.md

```diff
@@ -86,7 +86,7 @@ let adapter = instance
 
 Another thing to note is that `Adapter`s are locked to a specific backend. If you are on Windows and have 2 graphics cards you'll have at least 4 adapters available to use, 2 Vulkan and 2 DirectX.
 
-For more fields you can use to refine you're search [check out the docs](https://docs.rs/wgpu/0.10.1/wgpu/struct.Adapter.html).
+For more fields you can use to refine your search [check out the docs](https://docs.rs/wgpu/0.10.1/wgpu/struct.Adapter.html).
```

Comment:
your is correct here, as it's "refine your search" not "refine you are search"

Reply:
Yes, that's why I suggested the change in my pr. Maybe I'm missing something as I don't understand your change request.

Please excuse me as it's my first pull request and I'm not familiar with GitHub yet.

Reply:
Oh I read that wrong. I thought you were changing your -> you're